exporter.php

```diff
@@ -1,257 +1,42 @@
 <?php
-// phpcs:ignoreFile
 /**
- * Post Exporter Functionality
+ * Campaign Exporter Functionality
  *
  * @package P4MT
  */
 
 global $wpdb, $post;
-$defaults = [
-	'content'    => 'all',
-	'author'     => false,
-	'category'   => false,
-	'start_date' => false,
-	'end_date'   => false,
-	'status'     => false,
-];
-$args     = wp_parse_args( $post, $defaults );
+
+require_once 'exporter-helper.php';
+
 define( 'CMX_VERSION', '1.0' );
 $sitename = sanitize_key( get_bloginfo( 'name' ) );
 if ( ! empty( $sitename ) ) {
 	$sitename .= '.';
 }
-$filename = $sitename . gmdate( 'Ymdhis' ) . '.xml';
+
+// Added "P4-Campaign" word in XML filename to distinguish between normal WP export and P4 campaign export.
```

Comment:
Awesome, I was going to suggest changing the filename as indeed it was not very descriptive, but you already did that. Maybe we can go even further and add the campaign slug to the filename?

Reply:
Thanks Pieter :) I also think about adding slug in file name but we also provide option to export campaign in bulk.(select checkboxes and from top dropdown choose export option)

Reply:
Maybe we can check if they're exporting more than one campaign? And only add the campaign slug if they are exporting exactly one campaign?